README.md

```diff
@@ -13,12 +13,20 @@ is free, open-source software published under the GNU General Public License
 QUICKSTART:
 ===========
 
+To get a copy of the developer version (most recent version) of ESPResSo++, you can use git or docker.
+
 Using [docker](https://www.docker.com):
 ```
 # docker pull espressopp/espressopp
 # docker run -it espressopp/espressopp /bin/bash
 ```
 
+Using git:
```

Comment:
Maybe we should mention that docker is binary release (noting to compile, but not optimal performance), while git is a compile it yourself approach, which might lead to better performance.